products/eks.md

```diff
@@ -102,19 +114,25 @@ releases:
 > conformant for a predictable experience.
 
 Amazon EKS guarantees support for at least four production-ready versions of Kubernetes at any given
-time. A Kubernetes version is fully supported on EKS for 14 months after first being available on
-Amazon EKS. This is true even if upstream Kubernetes is no longer supporting a version available on
-Amazon EKS.
+time. A Kubernetes version is fully supported on EKS for 14 months as standard support and for up to
+26 months as exteneded support after first being available on Amazon EKS. This is true even if
+upstream Kubernetes is no longer supporting a version available on Amazon EKS.
 
 You can subscribe to upgrade notices (sent approximately 12 months after the Kubernetes version was
 released on Amazon EKS) on your [Personal Health Dashboard](https://aws.amazon.com/premiumsupport/technology/personal-health-dashboard/).
 The notice includes the end of support date, which is at least 60 days from the date of the notice.
 
+At the end of the of standard support for a Kubernetes version (starting from version 1.23), Amazon 
+EKS clusters running on EOL version will automatically enter extended support and Amazon apply an 
+additional charge per cluster hour for all clusters running an extended support version. If you 
+don’t want to be [automatically enrolled in extended support](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#extended-support-faqs), 
+you can upgrade your cluster to a Kubernetes version that’s in standard Amazon EKS support.
```

Comment:
```suggestion
Starting from version 1.23, Amazon EKS clusters running on EOL version will automatically enter extended support at the end of the of standard support and Amazon will apply an additional charge per cluster hour for all those clusters.
Users that don’t want to be [automatically enrolled in extended support](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#extended-support-faqs) must upgrade their cluster before the end of the standard support phase.
```

A more neutral formulation is preferred here (see https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide#tone-and-text).

Reply:
👍🏽